staff/francisco-fernandez/cv/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+
+    <section id="headercv">
+        <h1>CV Oscar Medina López</h1>
+        <address id="street">C/ Los Alamos 10, Barcelona</address>
+        <address id="telephone">Tlfn: 665436546</address>
+        <address id="mail">Mail: omedina@gmail.com</address>
+        <img id="foto">
+    </section>
+
+    <section id="experiencia">
+        <h2>EXPERIENCIA LABORAL</h2>
+        <table>
+            <tr>
+                <th>Periodo</th>
+                <th>Puesto</th>
+                <th>Descripción</th> 
+            </tr>
+            <tr>
+                <td>Actualmente</td>
+                <td>Camarero</td>
+                <td>Camarero en importante cadena de restaurantes de comida rápida</td>
+            </tr>
+            <tr>
+                <td>Enero 2017- Diciembre 2017</td>
+                <td>Encantador de perros</td>
+                <td>Adiestrando perros con problemas de comportamiento para particulares y empresas</td>
+            </tr>
+            <tr>
+                <td>Enero 2018- Marzo 2108</td>
+                <td>Masajista</td>
+                <td>Desarrollando labores de masajista en empresa de developers para que no estuvieran muy estresados pues es un trabajo muy estresante</td>
+            </tr>
+        </table>
+
+    </section>
+
+    <section id="formación">
+        <h2>FORMACIÓN ACADÉMICA</h2>
+            <table>
+                <tr>
+                    <th>Periodo</th>
+                    <th>Formación</th>
+                    <th>Descripción</th> 
+                </tr>
+                <tr>
+                    <td>Enero 2013 - Diciembre 2016</td>
+                    <td>Universidad de la Vida</td>
+                    <td>Aprendiendo todo lo que pude en la universidad mas importante del mundo</td>
+                </tr>
+                <tr>
+                    <td>2010-2013</td>
+                    <td>Bachillerato</td>
+                    <td>En el instituo de Los Alamos, en la especialidad ciencias puras</td>
+                </tr>
+                <tr>
+                    <td>1994-2010</td>
+                    <td>EGB</td>
+                    <td>En el colegio Maria Blanchard, se llamaba así entonces, EGB</td>
+                </tr>
+            </table>
```

Comment:
WARING TABLE DESING, PROCEED REMOVAL

Reply:
I dont understand this correction Aleix, what is wrong?

Reply:
you're using table desing, this is forbidden in html programming

Reply:
if you are using that table because you wanted to achieve the "table view" (projection, design), then aleix is totally right, and you should avoid using table for that purpose.

Reply:
Changed tables. Id's eliminated.